Library/Formula/folly.rb

```diff
@@ -0,0 +1,46 @@
+class Folly < Formula
+  desc "An open-source C++ library developed and used at Facebook."
```

Comment:
That's actually the "official" description of the project. I think the ambiguity is intentional, its supposed to feel like a grab bag of awesome C++ utilities which we use at FB.